### DISCUSSION TITLE
Bulk Chunk/Blocks editing api such like WorldEdit

### DISCUSSION THREAD

**Discussion by ilsubyeega** [Minestom/Minestom#1221] [Help] [answered]
WorldEdit has a feature (as you might know) that defining the regions then make those regions fill/replace/delete with blocks. (//set, //replace, //cut) etc.
Since I wanted to know because Minestom is a new, completly rewritten from scratch. I would like to know that Is it possible to use bulk/async block/chunk editing/replacement for blazing speed?

###### Ive searched with those keywords on discussions: worldedit, bulk, chunk

**Comment by MrlnHi** [comment]
Have a look at the `AbsoluteBlockBatch` (and also `RelativeBlockBatch`)

**Comment by MrGazdag** [comment, marked as answer]
What you are looking for is called Batches, you can read more about them on [the wiki](https://wiki.minestom.net/world/batch). (The method names have changed a bit since, iirc they’re called `setBlock` now)

**Reply by ilsubyeega** [reply]
Well, i was struggled for viewing the javadocs, however the official docs(or wiki) helped me! thanks!
